ci/create-pipeline.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+set -x
+
+TARGET=${TARGET:-local}
+
+fly validate-pipeline --config pipeline.yml
+
+fly --target ${TARGET} set-pipeline --config pipeline.yml --pipeline logsearch-healthcheck -n -l credentials.yml
```

Comment:
A sample credentials.yml might be handy for non-dta folks who see this

Reply:
good idea.

Reply:
good call.  i've pushed a example credentials file.